core/migrations/0022_auto_20240624_2118.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 5.0.6 on 2024-06-24 19:18
+from __future__ import unicode_literals
+
+from django.db import migrations, transaction
+from django.utils.text import slugify
+
+from core.models import Community
+
+
+def unique_slugify(community, slug):
```

Comment:
hmm instead of copying, you can create method create_slug() on Community and call it from the model in the loop below.

Reply:
Done.

Reply:
But...  "when using apps.get_model, you need to avoid calling methods that rely on the current model class directly, such as generate_unique_slug, because this method might not exist or might have a different implementation at the time of the migration." Maybe it should stay as it is?